test/Sentry.Log4Net.Tests/Sentry.Log4Net.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net8.0;net7.0;net6.0;net48</TargetFrameworks>
+    <TargetFrameworks>net9.0;net8.0;net48</TargetFrameworks>
```

Comment:
Do we have a net9.0 target for log4net? not sure it's worth adding. Just 8.0 is fine.
In tests we should match/have a target in test that covers